packages/vaadin-date-picker/test/basic.test.js

```diff
@@ -143,27 +143,23 @@ describe('basic features', () => {
     expect(spy.called).to.be.true;
   });
 
-  it('should not have label defined by default', () => {
-    expect(datepicker.label).to.be.undefined;
-  });
-
   it('should lead zeros correctly', () => {
     datepicker.value = '+000300-02-01';
-    expect(datepicker._inputElement.value).to.equal('2/1/0300');
+    expect(datepicker.inputElement.value).to.equal('2/1/0300');
```

Comment:
Let's update all the tests in this suite to also use `input` same as the others you already updated.

Reply:
Done